back/index.js

```diff
@@ -1,5 +1,5 @@
 module.exports = {
-  extends: ['./base.js', 'plugin:node/recommended'],
+  extends: ['../base.js', 'plugin:node/recommended'],
```

Comment:
I'm not sure moving this file here works ?  won't eslint expect you to do 

`extends: ['@strapi/eslint-config/back/index]` ?  

I had to keep back.js and create just back/typescript in the monorepo. Can you verify this :)

Reply:
I moved them around in both here and the monorepo and it works as expected for me, tweaking both the eslint configs `no-console: 2` has the expected updates in the codebases, so i think we're good...?

Reply:
Ok in that case I think it should be renamde to base or node to be more consistent

Reply:
so to be clear there would be: `@strapi/eslint-config/back/typescript` and `@strapi/eslint-config/back/node` or `@strapi/eslint-config/back/javascript`?

*changes would be reflected to the front configs too.

Reply:
Sounds a bit more explicit. Wdyt ?

Reply:
yeah i think `javascript` is clearer imo since `@strapi/eslint-config/front/node` doesn't make a lot of sense 👍🏼

Reply:
Done ✅